stable/insights-agent/templates/test/test-pod.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
     component: test-job
   annotations:
     "helm.sh/hook": post-install,post-upgrade
+    {{- with .Values.global.customWorkloadAnnotations }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
```

Comment:
This looks suspicious to me. I'm not sure what `toYaml` will do with an empty object. I'd try using `helm template` to see what this ends up looking like